.github/workflows/ruby.yml

```diff
@@ -31,4 +31,4 @@ jobs:
     - name: Run tests
       run: bin/rake
     - name: Run tests (with cocina)
-      run: SETTINGS__FEATURES__COCINA=true bin/rake spec
+      run: bin/rake spec
```

Comment:
Why is this setting being removed?

Reply:
d'oh, i was not looking closely enough at the CI steps, and missed that CI is running `bin/rake spec` both with and without this setting being true (i mistakenly thought we were only running specs with this override, and was trying to make default `rspec` behave on laptop the way i thought CI tests were behaving, hence the `test.yml` change, since i'd gotten a couple failures in CI that i didn't initially get locally, and realized that i had to run with `SETTINGS__FEATURES__COCINA=true` to get those same failures).

anyway, will undo this change, thanks for pointing this out.